src/main/resources/assets/twilightforest/lang/de_de.json

```diff
@@ -1,139 +1,201 @@
-  "advancement.twilightforest.twilight_hunter.desc": "Erlege etwas in der lokalen Tierwelt.",
-  "advancement.twilightforest.magic_map_focus": "Mit Feuer geschrieben",
-  "advancement.twilightforest.magic_map_focus.desc": "Stelle den magischen Kartenfokus mit einer Rabenfeder, Glowstonestaub und Fackelbeeren her.",
-  "advancement.twilightforest.kill_naga": "Schlangenjäger",
-  "advancement.twilightforest.kill_naga.desc": "Erlege den Naga in seinem Waldhof.",
-  "advancement.twilightforest.naga_armors": "Naga-Aufrüster",
-  "advancement.twilightforest.naga_armors.desc": "Stelle Nagaschuppenharnisch und Nagaschuppenbeinschutz her.",
-  "advancement.twilightforest.kill_lich": "Krieger des Lich",
-  "advancement.twilightforest.kill_lich.desc": "Töte den Lich an der Spitze seines Turms.",
-  "advancement.twilightforest.lich_scepters": "Zepterherrscher",
-  "advancement.twilightforest.lich_scepters.desc": "Erwerbe alle drei Zepter der Macht.",
+  "itemGroup.twilightforest": "Dämmerwald",
+
+  // FORTSCHRITTE
+  "advancement.twilightforest.root.desc": "Betrete das geheimnisvolle magische Waldgebiet: den Dämmerwald.",
+
+  // Allgemeiner Fortschritt
+  "advancement.twilightforest.hedge": "Kammerjäger",
+  "advancement.twilightforest.hedge.desc": "Besiege eine Spinne in einem Irrgarten.",
+
   "advancement.twilightforest.hill1": "Die Stiefel gehören mir!",
   "advancement.twilightforest.hill1.desc": "Besiege einen Rotkäppchenkobold in einem kleinen hohlen Hügel.",
   "advancement.twilightforest.hill2": "Was war das für ein Geräusch?",
   "advancement.twilightforest.hill2.desc": "Besiege einen Rotkäppchen-Pionier in einem mittleren hohlen Hügel.",
   "advancement.twilightforest.hill3": "Ich kann dich sehen",
-  "advancement.twilightforest.hill3.desc": "Besiege ein Twilightgespenst in einem großen hohlen Hügel.",
-  "advancement.twilightforest.hedge": "Kammerjäger",
-  "advancement.twilightforest.hedge.desc": "Besiege eine Spinne in einem Irrgarten.",
+  "advancement.twilightforest.hill3.desc": "Besiege ein Dämmergespenst in einem großen hohlen Hügel.",
+
+  "advancement.twilightforest.quest_ram": "Määhige Angelegenheit",
+  "advancement.twilightforest.quest_ram.desc": "Gieb dem Schafbock der Schatzsuche, was ihm fehlt.",
+  
+  // Karte
+
+  "advancement.twilightforest.magic_map_focus": "Mit Feuer geschrieben",
+  "advancement.twilightforest.magic_map_focus.desc": "Stelle den magischen Kartenfokus mit einer Rabenfeder, Glowstonestaub und Fackelbeeren her.",
+
   "advancement.twilightforest.magic_map": "Bis zum Horizont",
   "advancement.twilightforest.magic_map.desc": "Stelle eine magische Karte her.",
   "advancement.twilightforest.maze_map": "Nichts wie raus hier",
   "advancement.twilightforest.maze_map.desc": "Stelle die Labyrinthkarte her, nachdem du einen Labyrinthkartenfokus gefunden hast.",
   "advancement.twilightforest.ore_map": "Ist es das denn wert?",
   "advancement.twilightforest.ore_map.desc": "Stelle die Labyrinth-Erzkarte her.",
+
+  
```

Comment:
Lots of strange spacing throughout the file, much like this. You dont need this much spacing between blocks of stuff. See the en_us file, all the spacing there is neat and makes sense.